test/spec/directives/block.spec.js

```diff
@@ -47,6 +47,16 @@ describe('Directive: blocks', () => {
         expect($scope.lockedStaffProfileBlock).toBe(false);
       });
 
+      it('is false when currentRegistration is null', () => {
+        $scope.currentRegistration = null;
+        globalUserSpy.and.returnValue(null);
+
+        $compile('<name-question></name-question>')($scope);
+        $scope.$digest();
+
+        expect($scope.lockedStaffProfileBlock).toBe(false);
+      });
+
```

Comment:
@canac I'm not sure how these other 2 commits got in here. Do you think I just never pushed them from a long time ago?